ios/FirebaseContinue/FirebaseContinue/Classes/FCNContinue.h

```diff
@@ -18,32 +18,58 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+// A block (with an accompanying optional error) which is invoked on the main thread
+// when something the Firebase Continue library attempted to do asynchronously completed.
+// The error provided to the block will be non-nil if the it was unsuccessful.
+// See the library method documentation below for more details.
+typedef NSError* FCNContinueCompletionError;
+typedef void (^FCNContinueCompletionBlock)(__nullable FCNContinueCompletionError error);
```

Comment:
Unless you're targeting pre-Xcode 7, this should be _Nullable instead of __nullable.